plugins/inputs/kernel/sample.conf

```
@@ -1,4 +1,12 @@
-# Get kernel statistics from /proc/stat
+# Get various kernel statistics.
+# Currently collects:
+#   /proc/stat
+#   /proc/entropy
+#   /sys/kernel/mm/ksm (optional, if enabled)
+#
 # This plugin ONLY supports Linux
 [[inputs.kernel]]
-  # no configuration
+## Additional gather options
+## Possible options include:
+## * ksm - kernel same-page merging
+# collect = []
```

Please indent by two spaces
```suggestion
  ## Additional gather options
  ## Possible options include:
  ## * ksm - kernel same-page merging
  # collect = []
```